mfts/papermark
# Accessibility Advocate- Added close button in signup page

### Changes Implemented:
- Added a prominent close button ("X") to the signup page.
- Clicking the close button now dismisses the signup/login page and redirects the user to the landing/home page.

### Verification Steps:

1. Navigate to the signup/login page.
2. Click the close button ("X").
3. Verify that the signup page is dismissed and the user is redirected to the landing/home page.

### Result:
The close button functionality has been successfully implemented, enhancing user experience and accessibility.

### Status:
Completed and ready for review.
